var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -105,6 +109,8 @@ class Qt(Package):
     # depends_on("ogg", when='+multimedia')
 
     use_xcode = True
+    # parallel build fails on Sierra
+    parallel = False
```

Comment:
yes, it took like 3 hours in serial :-( it was during build stage. I am also not sure if this is the case for all macOS'es or not, @mathstuf did not have any issues on El Capitan with 5.7, if I am not mistaken.